data/json/monsters/zed_children.json

```diff
@@ -310,5 +310,45 @@
     "burn_into": "mon_zombie_child_scorched",
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BLEED", "GUILT", "POISON", "NO_BREATHE", "REVIVES", "BONES", "HARDTOSHOOT" ],
     "//": "GUILT because it still looks enough like a child to evoke pity"
+  },
+  {
+    "id": "mon_zombie_child_fungus",
+    "type": "MONSTER",
+    "name": "fungal child",
+    "name_plural": "fungal children",
+    "description": "It is hard to recognize a human child in this creature.  Most of its skin is covered in disgusting-looking mold, and many little fungi is seen growing through the cracks in its skin.",
```

Comment:
Two suggestions: consider moving away from the passive voice ("is covered," "is seen" – by whom?), and perhaps avoid using "skin" twice?

Here's one possibility:

> It is hard to recognize a human child in this creature. Disgusting-looking mold covers most of its skin. Many cracks perforate its body, with small fungal stalks poking through.

Reply:
Used your variant, thanks.